iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TieLine.java

```diff
@@ -289,4 +301,7 @@ default HalfLine setFictitious(boolean fictitious) {
      */
     HalfLine getHalf(Side side);
 
+    default BoundaryPoint getBoundaryPoint(Side side) {
+        throw new UnsupportedOperationException();
```

Comment:
I don't know, it would throw a null pointer exception (as every time we call it, it is to have a value and this object is supposed to never be null), wouldn't it be weird and more difficult to debug?